src/content_delegate.js

```diff
@@ -361,8 +361,19 @@ ContentDelegate.prototype.handleiQuerySummaryPage = async function () {
   if (!PACER.isIQuerySummaryURL(this.url)) {
     return;
   }
+  
+  if (PACER.hasTransactionReceipt() && !PACER.isCaseQueryAdvance()) {
+    // when a user tries to search a case using the first/middle/last name in the iquery form
+    // they might get redirected to a list of matching people before they reach the list of cases
+    // related to that name. Currently, we are not interested in this page where the user can
+    // choose a person.
+    //
+    // This if statement will end this function early if the user reaches this unwanted page.
```

Comment:
As explained in other comment, I suggest this goes into the isCaseQueryAdvance function.